css/style.css

```diff
@@ -0,0 +1,17 @@
+/** @format */
+#header {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+}
+
+@media (max-width: 1200px) {
```

Comment:
If you commented the first selector, shouldn't be good to comment the second one? Or do a reference that these selectors are grouped to apply a style to something.

Reply:
I would like to suggest you add a comment to every style or group of styles that are applied for a particular section on your code. Don't forget to follow [HTML and CSS best practices.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
I would like to suggest you add a comment to every style or group of styles that are applied for a particular section on your code. Don't forget to follow [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).